pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -227,10 +227,10 @@ public int getNicCount() {
     }
 
     private boolean isPhysicalNic(Path path) {
-        if (path.toString().contains("/virtual/")) {
-            return false;
-        }
         try {
+            if (path.toString().contains("/virtual/") || path.toRealPath().toString().contains("/virtual/")) {
```

Comment:
Maybe we don't need to check `path.toString().contains("/virtual/")` again after you add `path.toRealPath().toString().contains("/virtual/")`. 

Thank you, and let me know what you think.

Reply:
ok, resolve it